pom.xml

```diff
@@ -85,6 +85,12 @@
             </dependency>
 
             <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-api</artifactId>
+                <version>1.7.25</version>
```

Comment:
This could be extracted in a maven property

Reply:
Totally agree